index.html

```diff
@@ -123,21 +131,130 @@
       .nice();
 
     // AXES
 
+
+     var xAxis = d3.svg.axis()
+                   .scale(x)
+                   .orient("bottom");
+ 
+     svg.append("g")
+        .attr("class", "x axis")
+        .attr("transform", "translate(0," + (height) + ")")
+        .call(xAxis)
+       
+        .append("text")
+        .attr("x", 1998)
+        .attr("y", 2015)
+        .attr("dy", ".9em")
+        .style("text-anchor", "start")
+        .text("Year");
+ 
+     var yAxis = d3.svg.axis()
+                   .scale(y)
+                   .tickSize(width)
+                   .orient("right")
+                   .ticks(8);
+ 
+     svg.append("g")
+        .attr("class", "y axis")
+        .attr("transform","translate(0,0)")
+        .call(yAxis)
+ 
+        .append("text")
+        .attr("transform", "rotate(-90)")
+        .attr("x", 666)
+        .attr("y", 8700000000)
+        .attr("dy", ".9em")
+        .style("text-anchor", "start")
+        .text("Amount of Incentive");
+ 
+       var gy = svg.append("g")
+                   .attr("class", "y axis")
+                   .call(yAxis);
+ 
+       gy.selectAll("g").filter(function(a) {return a;})
+         .classed("minor", true);
+ 
+       gy.selectAll("text")
+         .attr("x", 666)
+         .attr("dy", 8700000000);
+
+      var formatNumber = d3.format(".0f");
+  
+      var xAxis = d3.svg.axis()
+                    .scale(x)
+                    .tickFormat(formatCurrency)
+                    .tickSize(height)
+                    .orient("bottom"); 
+  
+      svg.append("g")
+         .attr("class", "x axis")
+         //.attr("transform", "translate(0," + (height) + ")")
+         .call(xAxis)
+        
+         .append("text")
+         .attr("x", 250)
+         .attr("y", 580)
+         .style("text-anchor", "start")
+         .text("Year");
+  
+      var gx = svg.append("g")
+                  .attr("class", "x axis")
+                  .call(xAxis);
+  
+      gx.selectAll("g").filter(function(a) {return a;})
+        .classed("minor", true);
+  
+      var yAxis = d3.svg.axis()
+                    .scale(y)
+                    .tickSize(width)
+                    .orient("right");
+  
+      svg.append("g")
+         .attr("class", "y axis")
+         .attr("transform","translate(0,0)")
+         .call(yAxis)
+  
+         .append("text")
+         .attr("transform", "rotate(-90)")
+         .attr("x", -300)
+         .attr("y", -60)
+         .attr("dy", ".9em")
+         .style("text-anchor", "start")
+         .text("");
+  
+      var gy = svg.append("g")
+                  .attr("class", "y axis")
+                  .call(yAxis);
+  
+        gy.selectAll("g").filter(function(a) {return a;})
+          .classed("minor", true);
+  
+        gy.selectAll("text")
+          .attr("x", -30)
+          .attr("dy", -4);
+  
+        function formatCurrency(d) {
+          var s = formatNumber(d / 1e3);
+          return d === x.domain()[1]
+          ? "$";
```

Comment:
This line errors out. We went over this briefly, but ternary operators work like this:

`if ? then : else`

Where `if` is a statement that returns true or false, then is the value to be returned if the if statement is true, and `else` is the statement to be returned if the if statement is false.

You're missing the `:` and the `else` part.

I'll be avoiding these statements in class examples as I think they tend toward unnecessary terseness and complexity.